libs/branding/src/assets/daff-flower.svg

```diff
@@ -0,0 +1 @@
 offset="1" stop-color="#ffc810"/></linearGradient><linearGradient id="linear-gradient-2" x1="77.18" y1="63.54" x2="77.18" y2="32.09" xlink:href="#linear-gradient"/><linearGradient id="linear-gradient-3" x1="51.74" y1="91.71" x2="51.74" y2="37.71" xlink:href="#linear-gradient"/></defs><title>icon-logo</title><path class="cls-1" d="M6.46,71.19a36.78,36.78,0,0,1-5.31-1.65A1.82,1.82,0,0,1,0,67.76c.45-9,3.61-16.13,9.4-21.28,10.93-9.72,27.21-8.75,27.9-8.7A1.83,1.83,0,0,1,39,39.72c0,.33,4.92,1.75,4.86,5.22-.1,6.89-5.18,18.12-4.85,18.7a1.83,1.83,0,0,1-.66,2.49C37.67,66.52,22.89,75,6.46,71.19Z"/><path class="cls-2" d="M74.21,62.33a49.4,49.4,0,0,1-12.57-4.67A1.82,1.82,0,0,1,61,55.18c.25-.43-6.35-5.78-6.59-10.73-.26-5.12,5.83-9.86,5.8-10.37a1.81,1.81,0,0,1,1.69-1.93,52,52,0,0,1,17.82,2.39C91.83,38.54,98.86,47,100,59a1.81,1.81,0,0,1-1.13,1.86C89.93,64.45,81.05,63.9,74.21,62.33Z"/><path class="cls-3" d="M51.48,91.66a1.75,1.75,0,0,1-.87-.48c-11.13-11-13.76-22.17-14-29.63a34,34,0,0,1,2.21-13.86c.41-.92,2.74-5.69,3.66-5.29.43.19,2.86-4.37,6.6-4.67,4.29-.35,6.17,3.32,6.6,3,.81-.6,4.48,4.14,5.08,5a36,36,0,0,1,5.83,14.87c1.18,8-.21,19.86-13.54,30.75A1.84,1.84,0,0,1,51.48,91.66Z"/><path class="cls-4" d="M17.25,41.46,14.51,38a19.54,19.54,0,0,1-3.7-9.91L10.68,27l1-.6c11.77-7.22,23,.91,23.43,1.26l2.2,10.16Z"/><path class="cls-4" d="M59.42,26,61,25.05c3.4-2.11,12.36-6.43,21.91-1l1,.58-.09,1.14a22.61,22.61,0,0,1-1,5.32l-1.46,4s-16.66-3.6-19.38-1.91Z"/><path class="cls-5" d="M33.71,24.52c.38,5.22-.29,12.44,2.53,19a11.6,11.6,0,0,0,2.33,3.66c3.66,3.83,9,3.73,10,3.71a13.7,13.7,0,0,0,8.12-2.65,13.84,13.84,0,0,0,3.79-4.68c3.7-6.89,3.17-15.32,3-18.8C63.47,24.79,33.34,19.31,33.71,24.52Z"/><path class="cls-6" d="M27.16,5.38c1.48-1.91,6.07,1.54,9.5-.47C38.87,3.62,38.1,1.51,40.2.7S43.57,2,48.3,2.1a11.47,11.47,0,0,0,4.34-.46C54.53,1,55.28,0,56.46.36a4,4,0,0,1,2,1.94c.8,1.31.52,2,1.14,2.48s1.72.18,3.61-.07C63.69,4.65,69.32,4,70.37,6a3.8,3.8,0,0,1,.13,1.27c.08,2.73-.62,3.39-.2,4.35.35.8.93.55,3.15,1.94,1.7,1.07,2.85,1.81,3,2.94a3.35,3.35,0,0,1-.87,2.28C74.07,21,72.42,20.84,72,22.44A13.86,13.86,0,0,0,71.91,25c0,.87,0,1.17-.2,1.4-.41.41-1.14.1-2.68-.2a8.69,8.69,0,0,0-3.54-.27,5.52,5.52,0,0,0-2.75,1.74C61,29.49,61.33,31,60.34,31.27c-1.58.49-2.83-3.11-5.35-3-1.32,0-2.49,1.08-4.15,2.54-1.46,1.28-2.05,2.26-2.74,2.07s-.53-1.1-1.48-2.27a5,5,0,0,0-2.47-1.73c-.36-.1-1.13-.25-5,1.06-2.07.69-2.1.82-2.47.73-2.16-.48-1.82-4.82-4.08-5.48-.77-.23-.8.29-3.61.67s-3.33-.07-3.48-.27c-.33-.43-.25-1.06-.14-2.14s.39-1.29.27-1.74c-.26-.94-1.52-.85-2.87-1.87-.94-.71-2.29-2.22-2-3.55s2.39-2.33,3.34-2.74,1.68-.49,2.28-1.2a3.42,3.42,0,0,0,.73-1.94c.08-1.09-.39-1.44-.46-2.54A4.08,4.08,0,0,1,27.16,5.38Z"/><path class="cls-5" d="M35.26,20.38c-.48-.56.69-2.58,2.06-4.06,4-4.3,10.24-4.44,11.89-4.41s8.24.1,11.25,4.47c1.14,1.66,1.82,4.05,1.29,4.41-.36.25-.94-.69-2.29-1.06-2-.53-2.84.89-6.13,1.42a28.11,28.11,0,0,1-4.35.11,25.12,25.12,0,0,1-3.77-.17c-3-.47-3.15-1.44-5.24-1.53C37.29,19.43,35.74,20.94,35.26,20.38Z"/></svg>
```

Comment:
Run svgo.

Don't think these are needed either (after svgo runs). `id="Layer_"` and `data-name="Layer "`

Reply:
resolved in 8a7216e1c09a0aa96cd5a066151f7065b472f928 for 11% saved (.4KB). Not trememendous, but better than nothing!